components/engine_rocks/src/import.rs

```diff
@@ -167,10 +168,12 @@ mod tests {
             sst2.put(v.as_bytes(), v.as_bytes()).unwrap();
         }
         sst2.finish().unwrap();
+        let force_allow_write = false;
         db.ingest_external_file_cf(
             CF_DEFAULT,
             &[p1.to_str().unwrap(), p2.to_str().unwrap()],
             None,
+            force_allow_write,
```

Comment:
How about using this style for simpicity?
```suggestion
        db.ingest_external_file_cf(
            CF_DEFAULT,
            &[p1.to_str().unwrap(), p2.to_str().unwrap()],
            None,
            false,
```